### DISCUSSION TITLE
Reescale to custom range

### DISCUSSION THREAD

**Discussion by indominus-hub** [Kitware/trame#114] [Q&A]
Dear Tramers,

How can I set a feature like "Reescale to custom range" in my Trame application?

I can display the colourmap and the bar, but I would like to be able to adjust the range (max and min) to better identify differences in my model.

Thanks in advance for your answers. This tool is great!

**Comment by jourdain** [comment]
This looks like a VTK question which should be asked on https://discourse.vtk.org/. Unless you are using ParaView as backend?

But you will have to provide some code snippet so we have an idea of what you are doing and point you in the right direction.

**Comment by jourdain** [comment]
On ParaView it will be something like that

```python
from paraview import simple

lut = simple.GetColorTransferFunction('NameOfYourArray')
lut.RescaleTransferFunction(50.0, 100.0)
```
